src/components/Observations/details/index.tsx

```diff
@@ -100,16 +128,38 @@ export default function ObservationDetails(props: ObservationDetailsProps): JSX.
     }
   }, [zoneNames, searchProps.filtersProps]);
 
-  useEffect(() => {
-    if (observation) {
-      /**
-       * show info on due date and monitoring plot progress
-       */
-    }
-  }, [observation]);
-
   return (
     <DetailsPage title={title} plantingSiteId={plantingSiteId}>
+      {statusSummary && (
+        <Box marginBottom={3} display='flex' flexGrow={1}>
+          <Message
+            type='page'
+            priority='info'
+            title={strings.OBSERVATION_STATUS}
+            body={
+              <>
+                <Box marginBottom={3}>
+                  {
+                    strings.formatString(
+                      strings.OBSERVATIONS_REQUIRED_BY_DATE,
+                      statusSummary.pendingPlots.toString(),
+                      statusSummary.endDate
+                    ) as string
+                  }
+                </Box>
+                {
+                  strings.formatString(
```

Comment:
Should this be inside a `Typography` or `Box` component?

Reply:
Thanks for the review @charlie-tf . I can put this inside a box similar to the first message. The Message component has predefined styles for the text, won't need Typography here.